slack_sdk/oauth/installation_store/amazon_s3/__init__.py

```diff
@@ -252,7 +253,7 @@ def delete_bot(
                     )
                 except Exception as e:  # skipcq: PYL-W0703
                     message = f"Failed to find bot installation data for enterprise: {e_id}, team: {t_id}: {e}"
-                    self.logger.warning(message)
+                    raise SlackClientConfigurationError(message)
```

Comment:
For deletion method failures, we should raise an error than just writing warning-level logs